examples/index.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const http = require('http');
+const dispatcher = require('httpdispatcher');
+
+const opentracing = require('opentracing');
+const hawkularAPM = require('../index');
+
+const SERVER_PORT = 9000;
+
+opentracing.initGlobalTracer(new hawkularAPM.APMTracer({
+    recorder: new hawkularAPM.ConsoleRecorder(),
+    sampler: new hawkularAPM.AlwaysSampledSampler(),
+}));
+
+// /hello handler
+dispatcher.onGet('/hello', function(req, res) {
+    const serverSpan = opentracing.globalTracer().startSpan('hello', {
+        tags: {
+            'span.kind': 'serer',
```

Comment:
They are not mandatory but standard opentracing tags. They were added for demonstration how to add standard tags and they are not related how our producer/consumer will be determined.